packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1967,17 +1968,36 @@ export default class Meeting extends StatelessWebexPlugin {
           this.logger.error('Meeting:index#join --> Error joining the call on roap initialization, ', err);
           throw err;
         })
-        .then(() =>
-          this.roap
-            .sendRoapMediaRequest({
-              sdp: this.mediaProperties.peerConnection.sdp,
-              roapSeq: this.roapSeq,
-              meeting: this // or can pass meeting ID
-            })
-            .catch((err) => {
-              this.logger.error('Meeting:index#join --> Error joining the call on send roap media request, ', err);
-              throw err;
-            }))
+        .then(() => new Promise((resolve, reject) => {
+          let timerCount = 0;
+          // eslint-disable-next-line func-names
+          // eslint-disable-next-line prefer-arrow-callback
+          const joiningTimer = setInterval(function () {
```

Comment:
why couldn't you use an arrow function here?

Reply:
`this` object was undefined . SO i had to put the function with bind . Any other suggestions !?